app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
+        if type(config['enabled']) == str:
```

Comment:
Agreed. The implementation of the site_settings table stores everything as a string, and converts specific values to booleans (and back).

Tech debt?